perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -751,6 +770,20 @@ def Prepare(self, vm):
     logging.info('Set root password completed. Stdout:\n%s\nStderr:\n%s',
                  stdout, stderr)
 
+  def _validate_size(self, size):
+    """Validates flag for storage size, returns default if invalid.
+
+    Args:
+      size: (GB).
+    Returns:
+      size: (GB).
+
+    GCP supports storage sizes from 0 to 64TB currently.
+    """
+    if size < 0 or size > 64000:
+      size = 3334
+    return size
```

Comment:
If size is invalid we should throw an exception and fail the benchmark run. We can't silently change user's input to a default value....